tests/integration/suite/placement/quorum/insecure.go

```diff
@@ -124,7 +126,7 @@ func (i *insecure) Run(t *testing.T, ctx context.Context) {
 		if err != nil {
 			return false
 		}
-		err = stream.Send(&v1pb.Host{Id: "app-1"})
+		err = stream.Send(&v1pb.Host{Id: "app-1", Namespace: "default"})
```

Comment:
Please add tests which have namespace as an empty string. We should also include empty string namespace tests elsewhere if we don't already.

Reply:
Yeah, we do. We have one [here](https://github.com/dapr/dapr/blob/6e6db590a03d1db74a03da99866151caaee6a0ea/tests/integration/suite/placement/dissemination/namespaced.go#L200) specific to dissemination, and there two in an auth context [here](https://github.com/dapr/dapr/blob/d7b0259f05b791cbd0c378e6adbf3a15f2337e9e/tests/integration/suite/placement/authz/mtls.go#L115) and [here](https://github.com/dapr/dapr/blob/ac350666e30ad3cca042e36cee028233f61e5c91/tests/integration/suite/placement/authz/nomtls.go#L63).
I don't know if it makes sense to do the same for the quorum tests, since the namespace doesn't influence the leader election and dapr finding the right leader.